.github/workflows/marker-report-issue.yml

```diff
@@ -0,0 +1,79 @@
+name: Open marker report GH issue
+on:
+  # only manual for now
+  workflow_dispatch:
+  pull_request:  # TODO: remove before merge
+    branches:
+      - master
+
+jobs:
+  marker-report-issue:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+
+      - name: Checkout this repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        id: setup-python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Cache LocalStack community dependencies (venv)
+        uses: actions/cache@v3
+        with:
+          path: .venv
+          # we can re-use the venv from other actions, but shouldn't write one that may be used by others since we install additional deps
+          key: ${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('setup.cfg') }}-marker-report-issue
+          restore-keys: |
+            ${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('setup.cfg') }}
+
+      - name: Install dependencies
+        run: make install-dev
+
+      - name: Generate marker report
+        env:
+          PYTEST_ADDOPTS: "-p no:localstack.testing.pytest.fixtures -p no:localstack.testing.pytest.snapshot -p no:localstack.testing.pytest.filters -p no:localstack.testing.pytest.fixture_conflicts -p no:tests.fixtures -s --co --disable-warnings --marker-report --marker-report-path './target'"
+          MARKER_REPORT_PROJECT_NAME: localstack
+          MARKER_REPORT_COMMIT_SHA: ${{ github.sha }}
```

Comment:
It would be great to link https://github.com/localstack/localstack/pull/8979 in the PR description and mention in a comment that an existing plugin is used (I was quite confused where these params come from)...

Reply:
Haha yeah, the ` --marker-report --marker-report-path` is a bit hidden in the `PYTEST_ADDOPTS` :sweat_smile: